src/renderer/coremods/contextMenu/plaintextPatches.ts

```diff
@@ -0,0 +1,23 @@
+import type { PlaintextPatch } from "src/types";
+
+export default [
+  {
+    find: "♫ (つ｡◕‿‿◕｡)つ ♪",
+    replacements: [
+      {
+        match: /(function ..?\((.)\){)(var .,.=.\.navId)/,
```

Comment:
Use `\w+` instead of `.`
```suggestion
        match: /(function \w+\((\w+)\){)(var \w+,\w+=\w+\.navId)/,
```